extensions/doctrine/Numeric/Rand.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
Is er dan ook een goede manier om doctrine extensions te loaden per module? (zoals al eerder voorgesteld https://github.com/GEWIS/gewisweb/pull/101#discussion_r28846423) Ik kon dat nergens vinden, dus daarom heb ik het maar globaal gedaan wat niet zo netjes is.